components/Common/Button/index.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import { forwardRef } from 'react';
+import styles from './index.module.css';
+import type { FC, ButtonHTMLAttributes } from 'react';
+
+type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
+	kind?: 'primary' | 'secondary' | 'special';
+};
+
+const Button: FC<ButtonProps> = forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
If we use react 19, we don't need `forwardRef` anymore (something we can address later)

Reply:
good catch